From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Matthias Maennich <maennich@google.com>
Date: Thu, 22 Oct 2020 10:42:50 +0100
Subject: DROP ME: Out of tree difference by unclean upstream merge

Commit eb87047f0823 ("Merge c457cc800e89 ("Merge tag
'irq-core-2020-10-12' of
git://git.kernel.org/pub/scm/linux/kernel/git/tip/tip") into
android-mainline") introduced an out of tree difference that this patch
compensates.

Signed-off-by: Matthias Maennich <maennich@google.com>
Change-Id: Idc74a650a80f5b6eb165adff1633ced385e2e4dd
---
 drivers/irqchip/irq-gic-v3.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/irqchip/irq-gic-v3.c b/drivers/irqchip/irq-gic-v3.c
--- a/drivers/irqchip/irq-gic-v3.c
+++ b/drivers/irqchip/irq-gic-v3.c
@@ -675,7 +675,6 @@ static asmlinkage void __exception_irq_entry gic_handle_irq(struct pt_regs *regs
 
 	if (handle_domain_irq(gic_data.domain, irqnr, regs)) {
 		WARN_ONCE(true, "Unexpected interrupt received!\n");
-		log_abnormal_wakeup_reason("unexpected HW IRQ %u", irqnr);
 		gic_deactivate_unhandled(irqnr);
 	}
 }
